### ISSUE TITLE
How to change the piece images?

### ISSUE THREAD

**Issue by Michael-Aguirre-Merchan** [grzegorz103/ngx-chess-board#126]
Hi, it's me again.

I'm trying to change the chess pieces, but I don't really get where do I have to change them.

**Comment by grzegorz103** [comment]
Hi Michael

I broke custom icons a bit last times during refractoring. I published 2.1.4, in this it should be fine. Bump your package please. 
Try to make this:
Put icons in assets folder
```src/assets/icon.svg```

Now in .ts file create variable:
```
import { PieceIconInput } from 'ngx-chess-board';

...

  icons: PieceIconInput = {
    blackBishopUrl: '/assets/icon.svg',
    blackKingUrl: '',   // Put path to every icon like above
    blackKnightUrl: '',
    blackPawnUrl: '',
    blackQueenUrl: '',
    blackRookUrl: '',
    whiteBishopUrl: '',
    whiteKingUrl: '',
    whiteKnightUrl: '',
    whitePawnUrl: '',
    whiteQueenUrl: '',
    whiteRookUrl: ''
  };

```

Now in HTML file:
```
<ngx-chess-board [pieceIcons]="icons"></ngx-chess-board>
```

Let me know please if it helped

**Comment by Michael-Aguirre-Merchan** [comment]
It worked perfectly thank you
